src/System.Runtime.Extensions/src/System/AppDomain.cs

```diff
@@ -280,5 +280,144 @@ public void SetThreadPrincipal(IPrincipal principal)
                 _defaultPrincipal = principal;
             }
         }
+
+        public ObjectHandle CreateInstance(string assemblyName, string typeName)
+        {
+            // jit does not check for that, so we should do it ...
+            if (this == null)
+                throw new NullReferenceException();
+ 
+            if (assemblyName == null)
+                throw new ArgumentNullException("assemblyName");
+            Contract.EndContractBlock();
```

Comment:
You can delete these `EndContractBlock` annotations too.

Reply:
fixed